# The_Project
Bio-497 environmental genomics class project: Shelby and Jon

### To do list

__Shelby__
Shelby: What numbers in the jplace file correspond to fields? tree and placements
Shelby: what is named multiplicity? corresponds to the "nm" value in the jplace file
Shelby: draw a small tree and make a small jplace file and show how they correspond to eachother.
Shelby: Figure how to tell if placement on the tip vs internal 
Shelby: talk to eric about running on phylosift on Mac
Shelby: look into EDPL

__Jon__
Jon: What is the likelihood and why is negative. What is more likely vs less likely
Jon: what are the values that corresponde to the reference tree numbers
Jon: figure out how to change what information is displayed on the tree using archaeoptryx
Jon: How to find leaves in the tree section of the jplace file
Jon: Talk to statistican about hidden markov models and other math shit
Jon: Newick file format
Jon: Email Robin how much of our proposal paper can be shared if we are basically doing the exact same propject. what sort of reference can we include. 
Jon: look for figures to use. figure for jplace file, fat tree with placement labels, example of python code

### Project proposal paper:

__Sections__
Introduction
  *Background: talk metagenomics -> phylogenetic trees -> pipeline -> pplacer and fat trees
  *Questions/state of the field: No easy way to extract data. 
  *Primary goals: Write a script to extract a variety of data from jplace
  *Hypothesis: We will write a fucking mind blowing script null: it won't be fucking mind blowing or....analysis of internal vs leaves will alter tree interpretation.
Methods
  *Data type: jplace file/json
  *Annotation pipeline: phylosift
  *Statistics: summary statistics 
  *Software packages: phylosift
Expected Outcomes
  *What we expect to find: Fucking awesome script. extrapolate on how abundance data will influence interpretation
  *Mock figures: Psuedo code, concept map, phylogenetic fat tree vs normal tree, 
References
  *Ten references: uuhhhh





